### PR TITLE
feat: create founders page component

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Closes [#975](https://github.com/near/near-discovery/issues/975) [#976](https://github.com/near/near-discovery/issues/976)

[Demo](https://www.loom.com/share/d2db90ba0afa4e4793f29b3887f2924b?sid=02d35900-2f54-4da5-825d-79a76759caf5)

### CODE REVIEW

File: src/NearOrg/FoundersPage.jsx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+      ),
+    }}
+  />
+);
+
+const selfServeResources = [
+  {
+    name: "Content Repository",
+    description:
+      "As a founder, I need to be able to access a repository housing templates, guides and resources essential for my journey.",
```
Comment (charleslavon) [inline src/NearOrg/FoundersPage.jsx:180]: hey @LCCCunningham  what text would you like us to use as a short description under each of these links?
Screen Shot 2024-02-16 at 11 26 03 AM
 cc @shelegdmitriy

Comment (shelegdmitriy) [reply src/NearOrg/FoundersPage.jsx:180]: @charleslavon I think you put the wrong screenshot here

**Comment by shelegdmitriy** [comment]
Hey @ilblackdragon! Do we want to keep this descriptions or we can use something else? We can simply remove them
Знімок екрана 2024-02-20 о 19 19 00

The example without descriptions:
Знімок екрана 2024-02-20 о 16 15 27

File: src/NearOrg/FoundersPage.jsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+      "Engage with interactive technical and business support to answer specific questions that you will likely encounter while building your product and company.",
+    icon: "ph-github-logo",
+    url: "https://near-cp-alpha.vercel.app",
+    target: "_blank",
+  },
+  {
+    name: "Token Launch Resources",
+    description: "Discover detailed support to help you understand how to launch a token.",
+    icon: "ph-file-doc",
+    url: "https://drive.google.com/file/d/1-UkMbSj1GGqO4aa9LfkrTwVvscLYZkJ7/view",
```
Comment (charleslavon) [inline src/NearOrg/FoundersPage.jsx:197]: @shelegdmitriy this link is broken. We should use the same pattern as we use for the PDFs at `/papers`

File: src/NearOrg/FoundersPage.jsx [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    icon: "ph-article",
+    url: "https://www.hzn.xyz/hzn",
+    target: "_blank",
+  },
+  {
+    name: "Funding Opportunities & Founder & Builder Communities",
+    description:
+      "NEAR Horizon connects the Founders, Builders, and Investors building on NEAR",
+    icon: "ph-file-doc",
+    url: "https://docs.google.com/document/d/12DkxQddpZmtDAMWyD89igZPL141Z2x9-iOER2jCDyXc/edit#heading=h.g1c5xapiejrw",
```
Comment (charleslavon) [inline src/NearOrg/FoundersPage.jsx:216]: Replace this link with `https://near.org/ecosystem/get-funding`

**Pull request merged**